ecr-scripts/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  backend "s3" {
+    bucket         = "terraform-lambda-deploy-state"
+    key            = "terraform-backend/terraform.tfstate"
+    region         = "ap-southeast-3"
+    dynamodb_table = "terraform-lambda-deploy-locks"
+    encrypt        = true
```

Comment:
These are for the application to use, right? These are just samples and they get replaced?

Reply:
yes they get generated and replaced using the main.tf.tpl file in templates directory
![image](https://github.com/frgrisk/terraform-aws-turbo-deploy/assets/58250257/7ab2c0b7-4890-4f3a-a1c9-12f76b296c40)